### PR TITLE
Fix: toast message and rotation bug in edit profile

### LINKED ISSUE #697: Change toast text on successful login
**Describe the chore**

On successful login , toast says "success" instead of something like "logged in"
**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by KartikPatelOfficial**
Fixes #697
Changes: 
Fix toast text on successful login

### CODE REVIEW

**Comment by KartikPatelOfficial** [comment]
First timer here.

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt [hunk 1]
```diff
@@ -71,7 +71,7 @@ class LoginFragment : Fragment() {
         })
 
         loginActivityViewModel.loggedIn.observe(this, Observer {
-            Toast.makeText(context, "Success!", Toast.LENGTH_LONG).show()
+            Toast.makeText(context, "logged in", Toast.LENGTH_LONG).show()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Use string resource

**Comment by iamareebjamal** [comment]
Secondly, @Abhimanyu121 wanted to do this, so if he sends a PR, we are going to honor his over yours

File: app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt (hunk 1 above)
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Also capitalise the l in `logged in`

Comment (KartikPatelOfficial) [reply app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt]: Thank you.

**Comment by iamareebjamal** [comment]
What rotation bug? We accept one PR per issue and each PR must be linked to an issue

**Comment by Abhimanyu121** [comment]
I will make a PR today , changing toast notification text

**Comment by nikit19** [comment]
@KartikPatelOfficial I am closing this PR as #697 has already been solved

**Pull request closed without merging**